provider/oci/shapespecs.go

```diff
@@ -9,163 +9,268 @@ type ShapeSpec struct {
 	Cpus int
 	// Gpus is the number of GPUs available to this instance
 	Gpus int
-	// Memory is the amount of RAM available to the instance
+	// Memory is the amount of RAM available to the instance in MB
 	Memory int
-	Type   InstanceType
-	Tags   []string
+	// Bandwidth is the network bandwidth in Gbps. Where there are multiple physical NICs, the speed of the fastest is used.
+	Bandwidth float32
+	Type      InstanceType
```

Comment:
I know this was there already but generally if some attributes are comments, they all should be